CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 ## Unreleased
 
 * Add the `aria-invalid` attribute on inputs with errors.
+* Removed deprecated methods `default_input_size`. [@herminiotorres](https://github.com/herminiotorres)
```

Comment:
This doesn't need a changelog entry since the methods are still there, just not duplicated.